### ISSUE TITLE
Logging improvements: Sensor changes

### ISSUE THREAD

**Issue by tbaur** [node-alarm-dot-com/homebridge-node-alarm-dot-com#114]
A specific log level that only logs changes to sensors (and not API login re log level 3).

Examples -- motion activated/deactivated, door opened/closed, window opened/closed etc. and so on. 

While it's in the alarm.com app/web UI, this would be very helpful.

**Comment by chase9** [comment]
I'm thinking it would be best to just move the login call to the debug level. Thinking about it, it seems like it's not useful to know unless you're debugging something.

**Comment by chase9** [comment]
I just pushed a commit to the beta branch which moves some spammy calls to debug. Hope you can use log level 3 without being annoyed now!

**Comment by chase9** [comment]
I'll check this out and see which sensors are missing logging in the code. 

The issue of missed state changes has to do with limitations of this being a cloud polling plugin. If you both open and close a door within the time your plugin is not checking the cloud status, the plugin will just see that the door has kept the same state. The only way to fix this would be to lower the refresh rate super low (which I do not recommend!) or for us to implement web sockets.

**Comment by chase9** [comment]
So we are able to pull sensor history, but there is no way to inject history into Homekit. I think the way to acomplish what you're asking for would be to lower the polling time (something you can try yourself already) or implement websocket polling as mentioned in https://github.com/node-alarm-dot-com/node-alarm-dot-com/issues/27

**Comment by Omar-L** [comment]
You can certainly use ADC's WebSocket event stream to pick up events. It's near instant.

**Comment by chase9** [comment]
@Omar-L correct, which I've mentioned in the linked issue above. I would love a PR adding websocket support!

**Comment by chase9** [comment]
This has been added in the beta version of the plugin for people to test! Right now, it's opt-in in your config and only supports locks, sensors, and thermostats.

**Comment by chase9** [comment]
Closing as this has been merged into main.
